gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/exceptions/UserCannotBeCreatedException.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.service.exceptions;
+
+import io.gravitee.common.http.HttpStatusCode;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Exception thrown when a user cannot be created (e.g., user already exists during sign up/creation).
+ *
+ * @author GraviteeSource Team
+ */
+public class UserCannotBeCreatedException extends AbstractManagementException {
+
+    private final String source;
+    private final String userId;
+    private final String organizationId;
+
+    public UserCannotBeCreatedException(String source, String userId, String organizationId) {
+        this.source = source;
+        this.userId = userId;
+        this.organizationId = organizationId;
+    }
+
+    @Override
+    public int getHttpStatusCode() {
+        return HttpStatusCode.BAD_REQUEST_400;
+    }
+
+    @Override
+    public String getMessage() {
+        return "User cannot be created.";
+    }
+
+    @Override
+    public String getTechnicalCode() {
+        return "user.cannot.be.created";
```

Comment:
I think we usually have something like an object here. 

```suggestion
        return "user.invalid.creation";
```

Reply:
@jourdiw 
I added user.invalid to keep generic and created 1 exception class having both cases (it can be modified in future for multiple scenarios).